sqlbrite/src/main/java/com/squareup/sqlbrite/BriteContentResolver.java

```diff
@@ -120,9 +120,10 @@ public QueryObservable createQuery(@NonNull final Uri uri, @Nullable final Strin
       }
     };
     Observable<Query> queryObservable = Observable.create(subscribe) //
+        .onBackpressureLatest() // Guard against uncontrollable frequency of upstream emissions.
```

Comment:
just curious...why is this here instead of before the `observeOn`? I thought that since it was the scheduler that couldn't keep up with the emissions it would need the backpressure applied immediately before.